lib/crowbar/client/request/base.rb

```diff
@@ -50,7 +50,7 @@ def params
           when :post
             [
               method,
-              content.to_json
+              content
```

Comment:
I'm not sure if this is possible without mayor changes. The methods are chained and called after each other so we would have to add an extra param to not do the to_json. This sounds like that this would break the design concept of the crowbar-client.